fastcrypto/src/bls12381.rs

```diff
@@ -254,8 +254,11 @@ impl VerifyingKey for BLS12381PublicKey {
         }
         let mut rands: Vec<blst_scalar> = Vec::with_capacity(sigs.len());
 
+        // The first coefficient can safely be set to 1 (see https://github.com/MystenLabs/fastcrypto/issues/120)
+        rands.push(get_one());
```

Comment:
blst_scalar_from_uint32(1)

Reply:
oh, nvm, just saw that you are changing it to blst_scalar_from_uint64 which is the same

Reply:
That won't work. Like blst_scalar_from_uint64, which was used in [30132c0](https://github.com/MystenLabs/fastcrypto/pull/195/commits/30132c035466f13aebbc0020b56db3348412d9e7), the second argument is expected to be an array, so we still need to assume what the encoding is.